templates/confirmation/emailchangestatus_confirmation_email.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>Zulip</title>
+  </head>
+  <body>
+    <table width="80%" style="align:center; max-width:800px" align="center">
+      <tr>
+        <td style="font-size:16px; font-family:Helvetica;">
+          <p>Hi!
+          </p>
+
+          <p>Our records show that you requested a change to the email address
+          you use to sign in to Zulip. Please click the link below to activate
+          your new address.
+          <br />
+          <a href="{{ activate_url }}" style="color:#08c">{{ activate_url }}</a>
+          </p>
```

Comment:
This first paragraph should perhaps say:

"We received a request to change the email address for the Zulip account on {{ realm_uri }} associated with {{ old_email }} to {{ new_email }}.  If you'd like to confirm the change, please click this link:"